platform_channels/ios/Runner/AppDelegate.swift

```diff
@@ -45,7 +45,49 @@ import Flutter
         
         FlutterEventChannel(name: "eventChannelDemo", binaryMessenger: flutterViewController.binaryMessenger).setStreamHandler(AccelerometerStreamHandler())
         
+        var petList : [[String:String]] = []
+        
+        // A FlutterBasicMessageChannel for sending petList to Dart.
+        let stringCodecChannel = FlutterBasicMessageChannel(name: "stringCodecDemo", binaryMessenger: flutterViewController.binaryMessenger, codec: FlutterStringCodec.sharedInstance())
+        
+        // Registers a MessageHandler for FlutterBasicMessageChannel to receive pet details to be added in petList and send the it back to Dart using
```

Comment:
nit: "Registers a MessageHandler for FlutterBasicMessageChannel to receive pet details." is plenty of detail here. I would cut the rest of the comment.